### ISSUE TITLE
Make the table header fixed when people scroll

### ISSUE THREAD

**Issue by LeCoupa** [ratiw/vuetable-2#178]
Is there an easy way to make the table header fixed when it becomes hidden during scroll?

Thank you for your work :)

**Comment by ratiw** [comment]
@LeCoupa Since Vuetable is rendered out as normal HTML table, this is not possible.

**Comment by LeCoupa** [comment]
@ratiw Would you be interested by a PR for this feature?

**Comment by ratiw** [comment]
Yes, I'll be glad to look at it. :)

**Comment by galipmedia** [comment]
I would really like to see this feature, its one of my favourite things about datatables.

**Comment by LeCoupa** [comment]
I will probably work on it in a month.

**Comment by snyuan** [comment]
It would be a very important feature for me

**Comment by ChadTaljaardt** [comment]
+1

**Comment by rubinsh** [comment]
I've created a [pull request](https://github.com/ratiw/vuetable-2/pull/266) to address this issue.
Assuming the PR will be merged - once you'll define a height for the vuetable (via a new option called tableHeight) the header will become fixed.  
There is one caveat though - in order for the fixed header to behave correctly - the column widths need to be fixed.  
So I added also a "width" option to the column definitions.
